index.js

```diff
@@ -0,0 +1,46 @@
+'use strict';
+
+/**
+ * textarea-feedback Input Plugin
+ *
+ *
+ * A simple feedback textarea input type
+ */
+const validation = require('./lib/validation.js');
+const counter = require('./lib/charCounter.js');
+
+/**
+ * Single textarea-feedback Input Plugin
+ * @module textareaFeedbackInputPlugin
+ */
+module.exports = {
+  name: 'textarea-feedback',
+  description: 'A simple feedback textarea input type',
+  validation: {
+    textareaFeedbackValidation: validation,
+  },
+  scripts: {
+    characterCounter: counter,
+  },
+  inputs: {
+    textareaFeedback: {
```

Comment:
This is unique for the input plugin. You don't need to namespace this. If there's only the one `textarea` in there, it's OK to just call it `textarea`